fs.go

```diff
@@ -16,6 +15,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Marshaler is the interface implemented by types that
+// can marshal themselves into valid TOML.
+// TODO(carolynvs) Consider adding this to go-toml?
+type Marshaler interface {
```

Comment:
Thanks for noticing this! I had intended to put it in `go-toml` but that slipped my mind... 😊